invenio_app_rdm/theme/static/cc-icons/by-icon.svg

```diff
@@ -0,0 +1,52 @@
+<svg xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:cc="http://web.resource.org/cc/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:svg="http://www.w3.org/2000/svg" xmlns="http://www.w3.org/2000/svg" xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd" xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape" width="120" height="42" id="svg2759" sodipodi:version="0.32" inkscape:version="0.45+devel" version="1.0" sodipodi:docname="by.svg" inkscape:output_extension="org.inkscape.output.svg.inkscape">
```

Comment:
See the comment on moving the icons to a different folder.

In addition I would then name them ``cc-by.svg``.